### DISCUSSION TITLE
`magit-project-dispatch` for `project-switch-commands`

### DISCUSSION THREAD

**Discussion by mekeor** [magit/magit#5163] [Ideas]
Imagine you are in a buffer of project A but now you want to read the git-log of project B. Currently, I would use `magit-project-status` which `magit-extras` adds to `project-switch-commands` and `project-prefix-map` by default, and in the Magit status-buffer, I'd then press `l l` or so:

```
C-x p p project-b RET m l l
                        ^^^--- show log in magit-status-buffer
                      ^------- the key that magit-project-status is bound to by default
^^^^^^^----------------------- project-switch-project which will then let you choose a
                               project and then call any of the project-switch-commands
```

The here problem is primarily that showing a status-buffer might be slow. I suggest that Magit (perhaps `magit-extras` specifically?) provides a command that will also automatically be added to the `project-switch-commands` and `project-prefix-map` (perhaps bound to `M`) which dispatches a project-related Magit-transient (perhaps the default Magit status transient?) without needing to show a status-buffer.

**Comment by tarsius** [comment]
I've added such a command in f8ab44b2942e06a43475024a7c44bf9c45f9ae60. See the docstring for instructions.
